bitinformation_pipeline/bitinformation_pipeline.py

```diff
@@ -83,6 +84,153 @@ def get_keepbits(info_per_bit, information_content=0.99):
     return keepbits
 
 
+def plot_bitinformation(bitinfo):
+    from matplotlib import cm
+
+    nvars = len(bitinfo)
+    varnames = sorted(bitinfo.keys())
+
+    infbits_dict = get_keepbits(bitinfo, 0.99)
+    infbits100_dict = get_keepbits(bitinfo, 0.999999999)
+
+    ICnan = np.zeros((nvars, 64))
+    infbits = infbits100 = np.zeros(nvars)
+    ICnan[:, :] = np.nan
+    for v, var in enumerate(varnames):
+        ic = bitinfo[var]
+        ICnan[v, : len(ic)] = ic
+        infbits[v] = infbits_dict[var]
+        infbits100[v] = infbits100_dict[var]
+    ICnan = np.where(ICnan == 0, np.nan, ICnan)
+    ICcsum = np.nancumsum(ICnan, axis=1)
+
+    infbitsy = np.hstack([0, np.repeat(np.arange(1, nvars), 2), nvars])
+    infbitsx = np.repeat(infbits, 2)
+    infbitsx100 = np.repeat(infbits100, 2)
+
+    fig_height = np.max([4, 4 + (nvars - 10) * 0.2])  # auto adjust to nvars
+    fig, ax1 = plt.subplots(1, 1, figsize=(12, fig_height), sharey=True)
+    ax1.invert_yaxis()
+    ax1.set_box_aspect(1 / 32 * nvars)
+    plt.tight_layout(rect=[0.06, 0.18, 0.8, 0.98])
+    pos = ax1.get_position()
+    cax = fig.add_axes([pos.x0, 0.12, pos.x1 - pos.x0, 0.02])
+
+    ax1right = ax1.twinx()
+    ax1right.invert_yaxis()
+    ax1right.set_box_aspect(1 / 32 * nvars)
+
+    cmap = cm.get_cmap("copper_r").copy()
```

Comment:
could we take the one milan used in his paper?

Reply:
Turku is from Fabio Crameri's [Scientific Colour Maps](https://www.fabiocrameri.ch/colourmaps/), sure that's available for matplotlib somehow?

Reply:
thanks @milankl `pip install cmcrameri`

Reply:
Thanks @aaronspring and @milankl. Now the cmap is identical to the one used in the publication.